astroplan/plots/time_dependent.py

```diff
@@ -145,40 +147,49 @@ def plot_airmass(target, observer, time, ax=None, style_kwargs=None,
                       'to use a scalar. (Or maybe a list with length > 1?).',
                       PlotWarning)
 
-    # Calculate airmass
-    airmass = observer.altaz(time, target).secz
-    # Mask out nonsense airmasses
-    masked_airmass = np.ma.array(airmass, mask=airmass < 1)
+    if not isinstance(targets, list):
```

Comment:
Yeah, that was my thought, and I don't see someone make a generator to output targets but I suppose it could happen. I set it to `Sequence` for now but we could switch it.